helpers.cpp

```diff
@@ -19,8 +19,76 @@ void reflect(int height, int width, RGBTRIPLE **image) {
 // Definition for Blur Filter
 void blur(int height, int width, RGBTRIPLE **image) {
     // code to be written
+    
+    int sumBlue;
+    int sumGreen;
+    int sumRed;
+    int count;
+    //create a temporary table of colors to not alter the calculations
+    RGBTRIPLE tempImg[height][width];
+
+    for (int i = 0; i < width; i++)
+    {
+        for (int j = 0; j < height; j++)
+        {
+            sumBlue = 0;
+            sumGreen = 0;
+            sumRed = 0;
+            count = 0;
+
+            // sums values of the pixel and 8 neighboring ones, skips iteration if it goes outside the pic
+            for (int k = -1; k < 2; k++)
```

Comment:
```suggestion
            for (int k {-1}; k < 2; ++k)
```

- Use `{}` during initialisation and don't use `=`.
- Use prefix instead of postfix increment operator because postfix uses two cycles.